test/index.html

```diff
@@ -39,5 +39,13 @@ <h2>Navigation: Only a single navigation element should exist</h2>
     <section>
       <h2>HTML: html tag should have a lang attribute</h2>
     </section>
+    <section>
+      <h2>Description list: Direct children of a &lt;dl&gt; can only be &lt;dt&gt; and &lt;dd&gt;</h2>
+      <dl>
+        <dt>Hello, I'm dt element inside dl element</dt>
+        <dd>Hello, I'm dd element inside dl element</dd>
+        <p>Hello, I'm p(paragraph) element inside dl element. </p>
```

Comment:
We can probably just use plain text descriptions for these, we don't need to be super strict about stating the tag inside the test of the tag instead preferring readability (so long as it's obvious).

`Hello, I'm a paragraph element inside dl element.`